_data/jobs.yml

```diff
@@ -41,7 +41,7 @@
   location: State College, Pennsylvania
   name: ICDS Technical Director - Institute for Computational and Data Sciences (ICDS)
   posted: 2021-11-09
-  url: https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/ICDS-Technical-Director_REQ_0000020689
+  url: https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/
```

Comment:
If the original URL for a job doesn't work, it's probably better to remove the whole job - It seems like this new URL is not for a specific RSE job

Reply:
It works fine,
![job](https://user-images.githubusercontent.com/2630707/144484655-64e5b0b7-445d-4781-adb1-0b5d5ed1234b.png)
etc., just that the URL checker is *really* picky about certain things.